README_NEW.md

```diff
@@ -0,0 +1,654 @@
+
+```bash
+make anvil_deploy_eigen_contracts
+```
+
+You will also need to redeploy the MockStrategy & MockERC20 contracts:
+
+```bash
+make anvil_deploy_mock_strategy
+```
+
+#### Holesky
+
+The current EigenLayer contracts for Holesky are available in the [eigenlayer-holesky-contracts](https://github.com/Layr-Labs/eigenlayer-contracts/blob/testnet-holesky/script/configs/holesky/Holesky_current_deployment.config.json).
+
+#### Mainnet
+
+The current EigenLayer contracts for Mainnet are available in the [eigenlayer-mainnet-contracts](https://github.com/Layr-Labs/eigenlayer-contracts/blob/mainnet/script/configs/mainnet/Mainnet_current_deployment.config.json).
+
+### AlignedLayer
+
+#### Anvil
+
+When changing AlignedLayer contracts, the anvil state needs to be updated with:
+
+```bash
+make anvil_deploy_aligned_contracts
+```
+
+#### Holesky/Mainnet
+
+To deploy the contracts to Testnet/Mainnet, you will need to set environment variables in a `.env` file in the same directory as the deployment script (`contracts/scripts/`).
+
+The necessary environment variables are:
+
+| Variable Name                   | Description                                                           |
+|---------------------------------|-----------------------------------------------------------------------|
+| `RPC_URL`                       | The RPC URL of the network you want to deploy to.                     |
+| `PRIVATE_KEY`                   | The private key of the account you want to deploy the contracts with. |
+| `EXISTING_DEPLOYMENT_INFO_PATH` | The path to the file containing the deployment info about EigenLayer. |
+| `DEPLOY_CONFIG_PATH`            | The path to the deployment config file.                               |
+| `OUTPUT_PATH`                   | The path to the file where the deployment info will be saved.         |
+ 
+You can find an example `.env` file in [.env.example.holesky](contracts/scripts/.env.example.holesky)
+
+Then run the following command:
+
+```bash
+make deploy_aligned_contracts
+```
+
+You need to complete the `DEPLOY_CONFIG_PATH` file with the following information:
+
+```json
+{
+    "chainInfo": {
+      "chainId": "<chain_id>"
+    },
+    "permissions" : {
+      "owner": "<owner_address>",
+      "aggregator": "<aggregator_address>",
+      "upgrader": "<upgrader_address>",
+      "churner": "<churner_address>",
+      "ejector": "<ejector_address>",
+      "deployer": "<deployer_address>",
+      "initalPausedStatus": 0
+    },
+    "minimumStakes": [],  
+    "strategyWeights": [],
+    "operatorSetParams": [],
+    "uri": ""
+  }
+```
+
+You can find an example config file in `contracts/script/deploy/config/holesky/aligned.holesky.config.json`.
+
+### Bindings
+
+Also make sure to re-generate the Go smart contract bindings:
+```bash
+make bindings
+```
+
+
+## Run Devnet Locally with Anvil
+
+Start anvil with every relevant contract deployed with:
+
+```bash
+make anvil_start
```

Comment:
```suggestion
anvil_start_with_block_time
```

Reply:
Solved in [1c49ca6](https://github.com/yetanotherco/aligned_layer/pull/152/commits/1c49ca67a90eed08d341073d87e60ff4453c85bb)